Tasks/NpmAuthenticate/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "vsts-npm-authenticate-task",
+  "version": "0.0.1",
+  "description": "VSTS npm Authenticate Task",
+  "main": "npmauth.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/vsts-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/microsoft/vsts-tasks/issues"
+  },
+  "homepage": "https://github.com/microsoft/vsts-tasks#readme",
+  "dependencies": {
+    "vsts-task-lib": "2.0.6",
+    "npm-common": "../Common/npm-common"
```

Comment:
I saw this in the npm package.json as well. Are they being added by the build script or by npm? If npm is adding them we want to make sure they don't clash with whatever the build script is expecting.

Reply:
There is no adverse affect, AFAICT, to this.  Further, it is both supported in package.json and without it you've got an invisible dependency (bad).